src/main/java/world/inetum/realdolmen/inetumrealJobs/payload/request/SignupRequest.java

```diff
@@ -1,28 +1,32 @@
 package world.inetum.realdolmen.inetumrealJobs.payload.request;
 
+import world.inetum.realdolmen.inetumrealJobs.dtos.AddressDto;
+import world.inetum.realdolmen.inetumrealJobs.entities.enums.Gender;
+
 import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
 import javax.validation.constraints.NotNull;
-import java.util.Set;
+import java.time.LocalDate;
 
 public class SignupRequest {
-
-    private String username;
-
     @Email
     private String email;
-
-    private Set<String> role;
-
+    @NotBlank
+    private String role;
```

Comment:
Maybe use an enum, that way you make less mistakes

Reply:
We don't have any enum for the roles since they are represented as classes extending from Account

Reply:
Fixed